src/platform-integration/web/wasm.md

```diff
@@ -6,10 +6,6 @@ short-title: Wasm
 last-update: March 13, 2024
 ---
 
-Today, we are excited to celebrate a new form of Dart/JS interop and the future it will enable.
-Knowing our past, we are certain this is not the end of the journey, but just an exciting point in our history.
-Dart will continue to evolve 
```

Comment:
I'm not sure what was intended here. Seems to have been accidentally included?

Reply:
Agreed!